core/src/main/java/org/apache/brooklyn/core/entity/EntityAsserts.java

```diff
@@ -223,4 +225,14 @@ public void run() {
         });
     }
 
+    public static void assertServiceFailed(Entity test) {
+        assertFalse(test.sensors().get(SERVICE_UP), "Service should be down");
```

Comment:
I prefer the error message you'd get if you called `assertAttributeEquals(test, SERVICE_UP, false)`. That would also cope better with the sensor being null, giving us a nicer message.